javascript/javascript3/week2/style.css

```diff
@@ -0,0 +1,155 @@
+:root {
+    --clr--primary-200: #F3EED9;
+    --clr--primary-400: #824936;
+    
+    --clr--neutral-100: #FFF;
+    --clr--neutral-900: #222C2A;
+    
+    --ff-primary: 'Roboto', sans-serif;
+    }
+    
+    *{
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    }
+    
+    body{
+    font-family: var(--ff-primary);
+    font-weight: 400;
+    font-size: 1.3rem;
+    line-height: 1.6;
+    }
+    
+    .container {
+    margin-inline: auto;
+    width: min(90%, 70.5rem);
+    display: flex;
+    flex-direction: column;
+    }
+    
+    .text-center{
+    text-align: center;
+    }
+    
+    .bg-dark {
+    background-color: var(--clr--neutral-900);
+    color: var(--clr--neutral-100);
+    padding: 4rem 0;
+    display: flex;
+    height: 100vh;
+    }
+    
+    .bg-dark h2{
+    margin: 1.8rem 0 4rem 0;
```

Comment:
Using rem - nice 👍 
As a minor comment, I would suggest to only use multiples of 0.5 (or worst case 0.25), to keep the spacing consistent on the screen